context/brand-context/HISTORY.md

```diff
@@ -1,7 +1,12 @@
 # History
 
+## 18.0.0 (2021-12-03)
```

Comment:
If it's a new major version can you include `BREAKING:` before the bullets (see version 11 in this file)

Reply:
Although thinking about this more, this is an interesting one. Because in theory nothing should be broken after these changes. Which means this could be a minor version increment instead.

Reply:
Or did you intentionally bump major because there is potential for something to be broken but we are unsure? If that's the case maybe it's safest to declare it as breaking. 🤔

Reply:
> Breaking major change seems safest.

As alluded to in my comment regarding project specific CSS - let's go with breaking. 👍🏼